scripts/lib/wic/canned-wks/mkefidisk-multiboot.wks

```diff
@@ -10,4 +10,6 @@ part / --source rootfs --ondisk sda --fstype=ext4 --label platform --align 1024
 
 part swap --ondisk sda --size 44 --label swap1 --fstype=swap
 
+part /media --ondisk sda --size 787 --fstype=ext4 --label media --align 1024
```

Comment:
generate PARTUUID for this partition. Once you rebase you will see the changes and how to fix the PARTUUID for media partition.